etc/enpass.profile

```diff
@@ -19,10 +28,13 @@ include disable-xdg.inc
 
 include whitelist-var-common.inc
 
+# machine-id and nosound break audio notification functionality
+# comment both if you need that functionality or put 'ignore machine-id'
+# and 'ignore nosound' in your enpass.local
+
 caps.drop all
 machine-id
-net none
-no3d
```

Comment:
no3d removed?

Reply:
@SkewedZeppelin Nice catch! I forgot to re-add it after testing. Without no3d Enpass throws some errors about not finding libGL (which are cosmetic ones). I'll fix it, thanks.